Makefile

```diff
@@ -0,0 +1,14 @@
+all: fmt build
+
+fmt:
+	go fmt ...
```

Comment:
This fails for me:

```
$ make fmt
go fmt ...
/usr/lib/golang/src/go/build/zcgo.go
open /usr/lib/golang/src/go/build/zcgo.go.579441327: permission denied
exit status 2
make: *** [Makefile:4: fmt] Error 1
```

but if I call `go fmt` manually on the single files it works fine... Any idea what could be wrong?

Reply:
Ups, was a type. Should have been `go fmt ./...` and not `go fmt ...`.